python-client/giskard/core/suite.py

```diff
@@ -76,7 +76,15 @@ def to_mlflow(self, mlflow_client: MlflowClient = None, mlflow_run_id: str = Non
         return metrics
 
     def to_wandb(self, **kwargs) -> None:
-        """Log test suite results to the WandB run."""
+        """Log test-suite result to the WandB run.
+
+        Log current test-suite result in a table format to the active WandB run.
+
+        Parameters
+        ----------
+        **kwargs :
+            Additional keyword arguments to the active WandB run.
```

Comment:
could you please add this link https://docs.wandb.ai/ref/python/init saying something like:
```
**kwargs :
            Additional keyword arguments (see https://docs.wandb.ai/ref/python/init) to the active WandB run. 
```